packages/shared/src/components/checklist/ChecklistCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactElement, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { Card } from '../cards/Card';
+import MiniCloseIcon from '../icons/MiniClose';
+import { IconSize } from '../Icon';
+import { ChecklistStep } from './ChecklistStep';
+import { ChecklistAction, ChecklistCardProps } from '../../lib/checklist';
+
+const ChecklistCard = ({
+  className,
+  title,
+  description,
+  steps,
+  onRequestClose,
+}: ChecklistCardProps): ReactElement => {
+  const activeStep = useMemo(
+    () => steps.find((item) => !item.action.dateCompleted)?.action.type,
+    [steps],
+  );
+  const [checkedStep, setCheckedStep] = useState<string>(activeStep);
+  const isDone = useMemo(
+    () => steps.every((item) => !!item.action.dateCompleted),
+    [steps],
+  );
+
+  const onToggleStep = (action: ChecklistAction) => {
+    setCheckedStep((currentCheckedStep) => {
+      if (currentCheckedStep === action.type) {
+        return undefined;
+      }
+
+      return action.type;
+    });
+  };
+
+  return (
+    <div className={className}>
+      <Card className="p-0 rounded-14 border-cabbage-40 hover:!border-cabbage-40 w-[340px]">
```

Comment:
I don't think the color options here are good, please refrain from using colors directly as it doesn't include light/dark mode.
There should be a named version of this available. (Let me know if you can't find it)

Reply:
Yeah got it, It is clunky because in design colors are written like this and then finding those in theme is hell and they often mismatch 💦 . I'll try and remap to theme colors for sure and let you know.

<img width="304" alt="image" src="https://user-images.githubusercontent.com/9803078/235086572-5dde04fd-73cc-4d2c-a238-8f07c5fa857e.png">
<img width="299" alt="image" src="https://user-images.githubusercontent.com/9803078/235086630-732ce1e5-7294-4efa-9210-388cb74e74bc.png">